sass/directives/_carousels.scss

```diff
@@ -105,7 +105,10 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  text-align: center;
+  line-height: 28px;//this is for vertically centering arrows in ::after
```

Comment:
With our style base 1em = 16px
I would also recommend a less brittle approach for this, it's problematic if converting from `px` to `em` is all it takes to break this...
Try making the continue section on the carousel a flex-box and give everything `align-items: center`